### PR TITLE
Feature Tag Selection form inputFile to outputFile

### PR DESCRIPTION
**Pull request by temehi**
This will enable assignment of tags to an output file even if the format is different from that of the source format. 

Requested at #1935

### CODE REVIEW

**Comment by h-2** [comment]
@temehi can you rebase on the current develop? Then we will see if travis works again...

**Comment by h-2** [comment]
This looks good, but can you please add a test case, as well?

File: tests/seq_io/test_genbank.gbk [hunk 1]
```diff
@@ -0,0 +1,90 @@
+LOCUS       SCU49845     5028 bp    DNA             PLN       21-JUN-1999
```
Comment (h-2) [inline tests/seq_io/test_genbank.gbk]: why was this file added?

File: tests/seq_io/test_tag_select_intersect.h [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+#include <seqan/seq_io.h>
+#include <seqan/bam_io.h>
+
+// ---------------------------------------------------------------------------
+// Assign tags to an output file based on the format of input file.
+// ---------------------------------------------------------------------------
+
+void testTransferTag()
+{
+
```
Comment (h-2) [inline tests/seq_io/test_tag_select_intersect.h]: remove empty line

File: tests/seq_io/test_genbank.gbk (hunk 1 above)
Comment (temehi) [inline tests/seq_io/test_genbank.gbk]: To test the case of unavailable tag on the destination. Genbank format is in SeqInFormat  but not in SeqOutFormat. I wanted the SeqFileIn format to be detected automatically before trying to transfer the tag to the target format.

Comment (h-2) [inline tests/seq_io/test_genbank.gbk]: is there no other genbank file anywhere that you can use without adding a new one? If not, it is ok to add a new one!

Comment (temehi) [inline tests/seq_io/test_genbank.gbk]: Unfortunately no, there isn't!

**Comment by h-2** [comment]
@temehi AH, I just wanted to merge this and saw that there is no documentation, can you add dox with a minimal example? Sorry about the late notice.
